documentation/templates/bolt_pwsh_reference.md.erb

```diff
@@ -0,0 +1,45 @@
+# Bolt PowerShell Command Reference
```

Comment:
I think the generated markdown file needs to be added to the ditamap in order to be published: https://github.com/puppetlabs/bolt/blob/master/documentation/bolt.ditamap#L83 I think in the line below that would be a good place?

Reply:
added to the ditamap